assets/sass/components/settings/_googlesitekit-settings-notice.scss

```
@@ -209,6 +209,30 @@
 	}
 }
 
+.googlesitekit-settings-notice-offline-notice {
+	align-items: center;
+	bottom: 0;
+	gap: 15px;
```

I think this should be `$grid-gap-phone` for consistency with the notice padding:

```suggestion
	gap: $grid-gap-phone;
```